roles/openshift_node/tasks/upgrade/stop_services.yml

```diff
@@ -1,13 +1,19 @@
 ---
-- name: Stop node and openvswitch services
+- name: Stop node services
   service:
     name: "{{ item }}"
     state: stopped
   with_items:
-  - openvswitch
   - "{{ openshift_service_type }}-node"
   failed_when: false
 
+- name: Stop openvswitch service
+  service:
+    name: openvswitch
+    state: stopped
+  when: not openshift_use_nsx
+  failed_when: false
```

Comment:
That's not good. Why would it fail here? Why is this line necessary?

Reply:
I didnt want to change this as it was in the original task. I will remove it.

Reply:
Lets keep it - afaik this is for Atomic where the service would have a different name and its safe to skip this